SignalShareExtension/SAELoadViewController.swift

```diff
@@ -64,6 +65,14 @@ class SAELoadViewController: UIViewController {
         activityIndicator.startAnimating()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        // FIXME not until ready, and ideally before view appears to avoid any "loading flicker"
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 1) {
+            Logger.error("Simulating readiness...")
+            self.delegate?.shareExtensionIsReady()
```

Comment:
https://github.com/WhisperSystems/Signal-iOS/pull/2850 outlines my proposal here.  Basically `ShareViewController` like the main app will pop the loading view in its "ensure root view" method.

Reply:
Done!